runtime/jcl/common/java_lang_invoke_MethodHandleNatives.cpp

```diff
@@ -478,7 +478,7 @@ setCallSiteTargetImpl(J9VMThread *currentThread, jobject callsite, jobject targe
 		j9object_t callsiteObject = J9_JNI_UNWRAP_REFERENCE(callsite);
 		j9object_t targetObject = J9_JNI_UNWRAP_REFERENCE(target);
 
-		UDATA offset = (UDATA)vmFuncs->instanceFieldOffset(currentThread, J9VM_J9CLASS_FROM_HEAPCLASS(currentThread, callsiteObject), (U_8*)"target", strlen("target"), (U_8*)"Ljava/lang/invoke/MethodHandle;", strlen("Ljava/lang/invoke/MethodHandle;"), NULL, NULL, 0);
+		UDATA offset = (UDATA)vmFuncs->instanceFieldOffset(currentThread, J9OBJECT_CLAZZ(currentThread, callsiteObject), (U_8*)"target", strlen("target"), (U_8*)"Ljava/lang/invoke/MethodHandle;", strlen("Ljava/lang/invoke/MethodHandle;"), NULL, NULL, 0);
```

Comment:
We can spread it across multiple lines, which will make it easier to read: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#function-parameter-list-does-not-fit-on-one-line

Reply:
```
strlen("Ljava/lang/invoke/MethodHandle;")
```
The string is known during compile time. We can use `sizeof` instead of `strlen` since `sizeof` is evaluated at compile time. Some compilers replace `strlen` with `sizeof` as optimization. Not sure if it's happening in our case.

Reply:
There's a nice macro to help with this - `LITERAL_STRLEN(string)`

Reply:
updated to use suggested macro